### PR TITLE
Mayhem

### PR DESCRIPTION
**Pull request by ennamarie19**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "xpdf-4.03"]
```
Comment (xansec) [inline .gitmodules:1]: Is it necessary to change this file? Looks like the same module, just different protocol for access. I would not commit this file.

File: CMakeLists.txt [hunk 2]
```diff
@@ -4,6 +4,7 @@ project(pdfalto)
 set(CMAKE_CXX_STANDARD 11)
```
Comment (xansec) [inline CMakeLists.txt]: Is this necessary for the project or just the fuzzing target? If just for the fuzzing target, I might wrap this with and `#ifdef FUZZ` or something similar.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,33 @@
+#Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Please do not use git clone or `ldd`.

File: Mayhemfile [hunk 4]
```diff
@@ -0,0 +1,8 @@
+image: ghcr.io/ennamarie19/pdfalto-fuzz:latest
```
Comment (xansec) [inline Mayhemfile]: Please omit image field. Tests -> testsuite.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: CMakeLists.txt (hunk 2 above)
Comment (capuanob) [reply CMakeLists.txt]: I reset this.

File: .gitmodules (hunk 1 above)
Comment (capuanob) [reply .gitmodules:1]: Yes, the Docker build script was unable to pull the repository over SSH.

**Comment by capuanob** [comment]
@xansec Done!

**Comment by xansec** [comment]
One more question - the modified library files (`.a`), what is the purpose of those?

**Comment by ennamarie19** [comment]
@xansec Accidentally committed the libs directory. Fixed now!

**Comment by xansec** [comment]
LGTM!

**Pull request merged**